File: assignment 10 - C# practice problems/8.cs
//Write a c# program to reverse a given string. 

using System;
namespace MyNameSpace
{
    class Program8
    {
        public static void ReverseAStringDriverFunction()
        {

            Console.WriteLine("Enter a string: ");
            var inputString = Console.ReadLine();
            var split = inputString.ToCharArray();

            Array.Reverse(split);

            Console.WriteLine(string.Join(null, split));
        }
    }
}

File: assignment 10 - C# practice problems/js/8.js
//Write a JavaScript program to reverse a given stringInputing. 
function reverseAStringDriverFunction() {
    let stringInput = prompt("Enter a string");

    if (stringInput != null) {
        stringInput = stringInput.split("");
        stringInput = stringInput.reverse();
    }
    alert(stringInput.join(""));
}

reverseAStringDriverFunction()